src/Raven.Client/ServerWide/Operations/Logs/GetTrafficWatchConfiguration.cs

```diff
@@ -0,0 +1,87 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Net.Http;
+using Raven.Client.Documents.Changes;
+using Raven.Client.Documents.Conventions;
+using Raven.Client.Http;
+using Raven.Client.Json.Serialization;
+using Raven.Client.Util;
+using Sparrow.Json;
+using Sparrow.Logging;
+
+namespace Raven.Client.ServerWide.Operations.Logs;
```

Comment:
Please add a test for this.
You'll need to add a `PutTrafficWatchConfigurationCommand` as well.

Reply:
Done.